packages/forge/blocks/perplexity/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { createBlock } from "@typebot.io/forge";
+import { createChatCompletion } from "./actions/createChatCompletion";
+import { auth } from "./auth";
+import { PerplexityDarkLogo, PerplexityLightLogo } from "./logo";
+
+export const perplexityBlock = createBlock({
+  id: "perplexity",
+  name: "Perplexity",
+  tags: ["ai", "chat", "completion", "sonar", "sonar-reasoning"],
```

Comment:
```suggestion
  tags: ["ai", "chat", "completion", "sonar", "reasoning"],
```

keywords can be stacked up so no need to repeat sonar :)